### PR TITLE
Feature/water cooler zoom

### PR DESCRIPTION
**Pull request by zfhui**

### CODE REVIEW

File: app/controllers/zoom/water_cooler_controller.rb [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+              :id, 
+              :join_time
+            ]
+          ]
+        ]
+      )
+    end
+
+    def water_cooler_zoom_uuid
+      Blinkist::Config.get!('water_cooler_zoom_uuid')
```
Comment (parov) [inline app/controllers/zoom/water_cooler_controller.rb:57]: Is `water_cooler_zoom_uuid` set already?

File: app/controllers/zoom/water_cooler_controller.rb [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+    end
+
+    def slack_webhook_url
+      Blinkist::Config.get!('water_cooler_workflow_webhook')
```
Comment (parov) [inline app/controllers/zoom/water_cooler_controller.rb:61]: Is `water_cooler_workflow_webhook` set already?

**Review by parov — APPROVED** [review]
If the env variables are set, let's ship it!

File: app/controllers/zoom/water_cooler_controller.rb (hunk 1 above)
Comment (zfhui) [reply app/controllers/zoom/water_cooler_controller.rb:57]: Hi, no, none of it set, we still have to decide where that is coming from.

File: app/controllers/zoom/water_cooler_controller.rb (hunk 2 above)
Comment (zfhui) [reply app/controllers/zoom/water_cooler_controller.rb:61]: No, but we have the web hook URL, so we could add this to TF.

**Pull request merged**